### PR TITLE
Brush over docs

### PR DESCRIPTION
**Pull request by flxflx**
* Update concept diagram
* Reword some sections

### CODE REVIEW

**Comment by katexochen** [comment]
Change of yarn.lock/package-lock.json should be dropped.

File: docs/docs/examples/emojivoto.md [hunk 1]
```diff
@@ -154,25 +142,21 @@ The public facing frontend for voting uses the mesh certificate without client a
...
-## Voter's perspective: Verifying the ballot
+## Verifying the deployment as a user
 
-As voters, we want to verify the fairness and confidentiality of the deployment before
-deciding to vote. Regardless of the scale of our distributed deployment, Contrast only
-needs a single remote attestation step to verify the deployment. By doing remote attestation
-of the Coordinator, we transitively verify those systems the Coordinator has already attested
-or will attest in the future. Successful verification of the Coordinator means that
-we can be sure it will enforce the configured manifest.
+In different scenarios, users of an app may want to verify its security and identity before sharing data, e.g., before casting a vote. With Contrast, a user only needs a single remote-attestation step to verify the deployment - regardless of the size or scale of the deployment. Contrast is designed such that, by verifying the Coordinator, the user transitively verifies those systems the Coordinator has already verified or will verify in the future. Successful verification of the Coordinator means that
```
Comment (Freax13) [inline docs/docs/examples/emojivoto.md:147]: Please put each sentence on its own line. This makes futures diffs much more readable.

File: docs/static/img/concept.svg
Comment (Freax13) [inline docs/static/img/concept.svg:1]: https://tetris.fandom.com/wiki/Tetromino#Z :D

File: docs/docs/examples/emojivoto.md [hunk 2]
```diff
@@ -241,7 +225,7 @@ the list that already contains the `"web"` DNS entry:
       },
 ```
 
-### Update the manifest
+### Updatating the manifest
```
Comment (msanft) [inline docs/docs/examples/emojivoto.md:228]:
```suggestion
### Updating the manifest
```

File: docs/docs/examples/emojivoto.md [hunk 3]
```diff
@@ -17,37 +17,25 @@ votes to the frontend.
...
 ![emojivoto components topology](https://raw.githubusercontent.com/BuoyantIO/emojivoto/e490d5789086e75933a474b22f9723fbfa0b29ba/assets/emojivoto-topology.png)
 
-### Motivation
-
-Using a voting service, users' votes are considered highly sensitive data, as we require
-a secret ballot. Also, users are likely interested in the fairness of the ballot. For
-both requirements, we can use Confidential Computing and, specifically, workload attestation
-to prove to those interested in voting that the app is running in a protected environment
-where their votes are processed without leaking to the platform provider or workload owner.
-
```
Comment (burgerdev) [inline docs/docs/examples/emojivoto.md:27]: Maybe we could keep a shorter (or alternate) version of this somewhere? I think making the goal of verification explicit helps understanding the purpose of it.

File: docs/docs/getting-started/cluster-setup.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
 
 ## Prerequisites
 
-Install the latest version of the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/).
+* Install a recent version of the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/). Note that `apt install` will likely install an outdated version.
```
Comment (burgerdev) [inline docs/docs/getting-started/cluster-setup.md:5]: We know the exact version that's needed (2.44.1) from the [Azure docs](https://learn.microsoft.com/en-us/azure/aks/deploy-confidential-containers-default-policy#prerequisites). I'd also replace *`apt install`* with *version installed by your package manager*.

File: docs/docs/examples/emojivoto.md (hunk 3 above)
Comment (flxflx) [reply docs/docs/examples/emojivoto.md:27]: I added some explanation above.

**Comment by flxflx** [comment]
> Change of yarn.lock/package-lock.json should be dropped.

Done

**Review by burgerdev — APPROVED** [review]
lgtm

**Pull request merged**